examples/docker/00-hello-world/worker.js

```diff
@@ -0,0 +1,15 @@
+const Queue = require('bee-queue');
+
+// Core worker logic
+const workerProcess = (job, done) => {
+  done(null, {greeting: 'Hello ' + job.data.name + '!'});
+};
+
+const go = () => {
+  return new Queue('hello-world', {redis: {host: 'redis'}, isWorker: true})
+    .ready()
+    .then((queue) => queue.process(workerProcess))
+    .catch(console.error);
```

Comment:
I'd recommend failing hard instead of just logging the error.
```suggestion
    .catch((err) =>
      process.nextTick(() => {
        throw err;
      });
    );
```